includes/templates/story-templates/fandom-cta.html

```diff
@@ -4,7 +4,7 @@
     <h2 style="color:#ffffff;text-align:center;display:flex" class="wp-block-amp-amp-story-text has-text-color"><amp-fit-text layout="flex-item" class="amp-text-content">Check your knowledge in Game of Thrones fan-made Trivia!</amp-fit-text></h2>
     <!-- /wp:amp/amp-story-text -->
 
-    <!-- wp:amp/amp-story-cta {"opacity":25,"customBackgroundColor":"#000000"} -->
-    <amp-story-cta-layer class="wp-block-amp-amp-story-cta aligncenter" id="f165e35a-5b53-49a6-88ea-c66a8bc1be61"><a class="amp-block-story-cta__link amp-text-content has-text-color has-background" style="background-color:rgba(0, 0, 0, 0.25);color:#ffffff;text-align:center"><strong>S</strong>tart Quiz</a></amp-story-cta-layer>
+    <!-- wp:amp/amp-story-cta {"opacity":25,"customBackgroundColor":"#000000","btnPositionLeft":30.18} -->
```

Comment:
Story template version would need to be bumped

Reply:
Thanks! Done.